frontend/config/index.ts

```diff
@@ -0,0 +1,4 @@
+import dayjs from 'dayjs'
+import advancedFormat from 'dayjs/plugin/advancedFormat'
```

Comment:
[`advancedFormat`](https://github.com/iamkun/dayjs#plugin) allows formatting with `Do`. Other formats [here](https://day.js.org/docs/en/parse/string-format#list-of-all-available-parsing-tokens)